Src/libCZI_UnitTests/test_writer.cpp

```diff
@@ -2144,3 +2144,92 @@ TEST(CziWriter, TryAddingDuplicateAttachmentToCziWriterAndExpectError)
     // now, try to add it a second time
     EXPECT_THROW(writer->SyncAddAttachment(add_attachment_info), LibCZIException);
 }
+
+TEST(CziWriter, TryAddingDuplicateSubBlocksToCziWriterAndExpectError)
+{
+    const auto writer = CreateCZIWriter();
+    const auto output_stream = make_shared<CMemOutputStream>(0);
+
+    const auto czi_writer_info = std::make_shared<libCZI::CCziWriterInfo>(GUID{ 0, 0, 0, {0, 0, 0, 0, 0, 0, 0, 0} });
+
+    writer->Create(output_stream, czi_writer_info);
+
+    // now add two subblocks (does not really matter, though)
```

Comment:
I don't understand the "does not really matter, through" remark

Reply:
right, thanks, I guess this comment was misplaced (copy-paste error I reckon). Fixed.